kratos/python/add_kratos_parameters_to_python.cpp

```diff
@@ -18,34 +17,32 @@
 // Project includes
 #include "includes/define_python.h"
 #include "includes/kratos_parameters.h"
-
 #include "add_kratos_parameters_to_python.h"
 
 namespace Kratos {
 
 namespace Python {
 
-
 pybind11::list items(Parameters const& self)
 {
     pybind11::list t;
-    for(Parameters::const_iterator it=self.begin(); it!=self.end(); ++it)
+    for(auto it=self.begin(); it!=self.end(); ++it)
```

Comment:
does `auto` determine the correct type here? => `iterator` or `const_iterator`?

With iterators I am usually a bit unsure

@roigcarlo do you know?

Reply:
The self is const, so the auto can be only a const iterator.. In any can be changed
```suggestion
    for(Parameters::const_iterator it=self.begin(); it!=self.end(); ++it)
```

Reply:
Reverted @philbucher